### ISSUE TITLE
Segmentation fault CircleCI

### ISSUE THREAD

**Issue by genbit** [edib-tool/mix-edib#3]
Trying to run it on CircleCI
getting this error.

exrm is in project dependency and I'm able to build it locally.

``` web/views/layout_view.ex
| web/views/page_view.ex
| web/views/user_view.ex
| sent 242,899 bytes  received 2,280 bytes  490,358.00 bytes/sec
| total size is 234,034  speedup is 0.95
| if [ -d /root/ssh ]; then \
|   mkdir /root/.ssh && \
|   rsync -av --no-owner /root/ssh/ /root/.ssh; \
| fi
| make[1]: Leaving directory '/build'
| make -f edib/edib.mk
| make[1]: Entering directory '/build'
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| make -f edib/check-exrm.mk
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| Segmentation fault (core dumped)
| make[2]: Entering directory '/build'
| Segmentation fault (core dumped)
| edib/check-exrm.mk:12: *** The project as no 'exrm' dependency. Cannot proceed further.  Stop.
| make[2]: Leaving directory '/build'
| make[1]: *** [exrm] Error 2
| edib/edib.mk:28: recipe for target 'exrm' failed
| make[1]: Leaving directory '/build'
| edib/Makefile:6: recipe for target 'edib' failed
| make: *** [edib] Error 2
```

**Comment by genbit** [comment]
Looks like circleci doesn’t allow removing intermediate containers. So I tried to run with mix edib --no-rm. It didn't help. But updating circleci to use docker 1.10 solves the issue:
https://discuss.circleci.com/t/docker-1-10-0-is-available-beta/2100

**Comment by asaaki** [comment]
Yes, the segmentation faults were always annoying and I never got rid of them on CircleCI until now, see my notes on https://github.com/edib-tool/edib-app-test-ci. Will test docker 1.10 for this.

**Comment by genbit** [comment]
I didn’t have segmentation fault with 1.10. But have now another issue:

```
| cat /build/tools/artifact.cfg.template | \
| sed "s@TPL_TARBALL_FILE@20160502_045728/myapp-0.0.1.tar.gz@g" | \
| sed "s@TPL_IMAGE_NAME@frontendyteam/myapp@g" | \
| sed "s@TPL_IMAGE_TAG@0.0.1@g" | \
| sed "s@TPL_IMAGE_SETTINGS@--change 'CMD trap exit TERM; /app/bin/myapp foreground \& wait'@g" \
| > /stage/tarballs/artifact.cfg
| make[2]: Leaving directory '/build'
| make[1]: Leaving directory '/build'
==> Artifact successfully created.
==> Creating docker image ...
| $> cat /home/ubuntu/myapp/tarballs/20160502_045728/myapp-0.0.1.tar.gz | docker import --change 'CMD trap exit TERM; /app/bin/myapp foreground & wait' - frontendyteam/myapp:0.0.1
| Error response from daemon: ApplyLayer exit status 1 stdout:  stderr: archive/tar: invalid tar header
| cat: write error: Broken pipe
==> An error happened!
==> Reason: Image command 'cat /home/ubuntu/myapp/tarballs/20160502_045728/myapp-0.0.1.tar.gz | docker import --change 'CMD trap exit TERM; /app/bin/myapp foreground & wait' - frontendyteam/myapp:0.0.1' failed
```

**Comment by asaaki** [comment]
This one is a known one, see #1 — fix is prepared but not released, I initially wanted to have feedback, but will release it soon anyway (worked for me).

**Comment by genbit** [comment]
Thanks! gunzip -c fixed the issue.

**Comment by asaaki** [comment]
Will release a more up-to-date version of the mix task soon.

**Comment by asaaki** [comment]
I've released version `0.6.1` which includes the `gunzip` approach.
Happy CircleCI building! 💖
